frontend/ad-mission/src/components/Payment.vue

```diff
@@ -0,0 +1,125 @@
+<template>
+  <div class="form-container">
+    <h1 class="form-title">Ad Mission</h1>
+    <form @submit.prevent="submitForm">
+      <div>
+        <label for="cardNumber">Card Number:</label>
+        <input type="text" id="cardNumber" v-model="cardNumber" required>
+      </div>
+      <div>
+        <label for="expiry">Expiry Date:</label>
+        <input type="text" id="expiry" placeholder="MM/YYYY" v-model="expiry" required>
+      </div>
+      <div>
+        <label for="cvv">CVV:</label>
+        <input type="text" id="cvv" v-model="cvv" required>
+      </div>
+      <div>
+        <label for="name">Cardholder Name:</label>
+        <input type="text" id="name" v-model="cardholderName" required>
+      </div>
+      <button type="submit">Submit Payment</button>
+    </form>
+  </div>
+</template>
+
+
+<script>
+export default {
+  data() {
+    return {
+      cardNumber: '',
+      expiry: '',
+      cvv: '',
+      cardholderName: ''
+    };
+  },
+  methods: {
+    submitForm() {
+      // Input validation
+      if (!/^\d{16}$/.test(this.cardNumber)) {
+        alert('Please enter a valid 16-digit card number.');
+        return;
+      }
+
+      if (!/\d{2}\/\d{4}/.test(this.expiry)) {
+        alert('Please enter a valid expiry date in MM/YYYY format.');
+        return;
+      }
+
+      if (!/^\d{3}$/.test(this.cvv)) {
+        alert('Please enter a valid 3-digit CVV.');
+        return;
+      }
+
+      // Form submission logic
```

Comment:
We need to save all these information to the database. You have to make a post api call to store this information in the database. As of now you are only console logging form submitted. All the data is then lost.
Also make sure to give user a message in UI rather than console logging the success info.